content/docs/api-reference/get_params.md

```diff
@@ -0,0 +1,90 @@
+# dvc.api.get_params()
+
+Returns all the <abbr>params</abbr> associated with the given
+<abbr>stage</abbr>.
+
+```py
+def get_params(stage: str, flatten: bool = False) -> Dict:
+```
+
+#### Usage:
+
+```py
+import dvc.api
+
+params = dvc.api.get_params("train")
+```
+
+## Description
+
+Retrieves the `dvc params` associated with a given `dvc.yaml` stage.
+
+Only the keys indicated in the
+[`params` section of the stage](/doc/user-guide/project-structure/pipelines-files#stage-entries)
+will be returned.
+
+When multiple `dvc params` files are associated with a stage, this function
+takes care of aggregating and selecting all the relevant keys, regardless of the
+file format used.
+
+## Parameters
+
+- **`stage`** (required) - Name of the dvc.yaml stage to get params from.
+
+- `flatten` - Whether to return flattened dict or not. _Default_: False.
```

Comment:
how essential it is to have this option? people could use some lib to do this if they want, wdyt?

Reply:
IMO, it's not essential to have it exposed. 

I exposed it because the default behavior of this public API (flatten `False`) is different from the default of the internal `Repo` API (flatten `True`). 

However, given that we don't document the `Repo` API at all this might not matter at all.